index.html

```diff
@@ -7,10 +7,39 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Portfolio Site</title>
     <link rel="stylesheet" href="./style.css">
-    <link rel="icon" href="./favicon.ico" type="image/x-icon">
+    <link href='https://fonts.googleapis.com/css?family=Playfair Display' rel='stylesheet'>
+    <link href="https://fonts.cdnfonts.com/css/poppins" rel="stylesheet">
 </head>
 
 <body>
+    <nav>
+        <div class="menu">
+            <div>
+                <img id="triangle" src="./assets/img/triangle.png" alt="Triangle">
+                <img id="square" src="./assets/img/square.png" alt="square">
+            </div>
```

Comment:
- [ ] Kindly remove the images and try implementing them using CSS and the `background-image` property. If an image is not part of the content (aesthetic reasons) use CSS background-image. [Check this link](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) for more info.